bundles/org.openhab.binding.wemo/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -17,7 +17,10 @@
 				<label>Unique Device Name</label>
 				<description>The UDN identifies the WeMo Device</description>
 			</parameter>
-
+			<parameter name="ipaddress" type="text" required="false">
+				<label>IP-Address</label>
```

Comment:
"IP address" (no dash and probably lower-case **a**.

Reply:
Done